tests/insert-pragma/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,87 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`module-with-late-docblock.js 1`] = `
+function foo(bar) { 
+  return bar + 3 + 4;
+}
+
+/**
+ * Some notes that should not be appended to
+ */ 
+const fruit = 'tomatoes';
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/** @format */
```

Comment:
I think we should add an extra `\n` here.

Reply:
in:
```js
/**
 * @flow
 * @snails
 */
function foo(bar) { 
  return bar + 3 + 4;
}
```

out:
```js
/**
 * @flow
 * @snails
 */

function foo(bar) { 
  return bar + 3 + 4;
}
```

I can't imagine people being upset about the newline, so i'll add it.

Reply:
implemented in: https://github.com/prettier/prettier/pull/2865/commits/f5193c894d679ca30a548e2f0fb19151acc628c0